azure_functions_worker/bindings/meta.py

```diff
@@ -70,6 +69,17 @@ def from_incoming_proto(
     else:
         metadata = {}
 
+    pb_type = pb.WhichOneof('rpc_data')
```

Comment:
Previously, we use `pb.data` but now we use `pb.WhichOneof('rpc_data')`. I saw you already written an if-check below to ensure the backward compatibility. Just want to double check if the host now sends an `rpc_data` field even when `SHARED_MEMORY_DATA_TRANSFER` is turned off.

Reply:
It will always send rpc_data - if using shared memory, the rpc_data.rpc_shared_memory will be populated by the host and when not using shared memory, the regular rpc_data.data field will be populated.